### PR TITLE
Added wraping restrictions

### PR DESCRIPTION
**Pull request by EPuncker**
Added:
- Can't move wrap items outside house tile
- Can't wrap/unwrap if you are not house owner
- Can't move carpet items into door tiles, to prevent this:

2020-04-06 23_34_26-Tibia - Perflex
(the item become unreachable/unwrapable)

follow up to https://github.com/otland/forgottenserver/commit/3cccb4564fc7edefdf8efed03fb3db1e85607d64

and yes they are on real tibia and I tested the changes on my repo, but hope someone can test it on clean TFS as well

credits: 
change on [onMoveItem to oen432](https://otland.net/threads/tfs-1-3-help-with-script-logic.269582/#post-2600402)
change on [onWrapItem to Zothion](https://otland.net/threads/event-antitrash-antitheft-system-for-houses-tfs-1-x.237075/page-2#post-2416471)
sarah wesker to moving code to carpet script instead

### CODE REVIEW

**Comment by dudantas** [comment]
I tested it and when I try to move an item on the floor it returns "sorry, not possible."

File: data/events/scripts/player.lua [hunk 1]
```diff
@@ -86,6 +86,13 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	if fromPosition.x ~= CONTAINER_POSITION and toPosition.x ~= CONTAINER_POSITION then
+		if not Tile(toPosition):getHouse() and item:getAttribute("wrapid") then
```
Comment (dudantas) [inline data/events/scripts/player.lua:90]: I tested it and "not" prevents the player from moving the item.

Comment (EPuncker) [reply data/events/scripts/player.lua:90]: yes, you shouldn't be able to move an item that have wrapit to a normal tile, only house tile

Comment (dudantas) [reply data/events/scripts/player.lua:90]: The problem is that I tested it with legs and armor.

Comment (EPuncker) [reply data/events/scripts/player.lua]: okay fixed, thanks

**Comment by omarcopires** [comment]
Tested, the code works perfectly.

File: data/scripts/actions/others/carpets.lua [hunk 2]
```diff
@@ -19,24 +19,32 @@ function carpets.onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	if not carpet then
 		return false
 	end
+	if fromPosition.x == CONTAINER_POSITION then
+		player:sendTextMessage(MESSAGE_STATUS_SMALL, "Put the item on the floor first.")
+		return true
+		end
```
Comment (luanluciano93) [inline data/scripts/actions/others/carpets.lua:25]: indentation

**Pull request merged**